website/scripts/neo4j-wdc-2.js

```diff
@@ -1,328 +1,354 @@
 var config = {};
 
 function executeNeoQueries(config, username, password, mode, fnResults, fnReason) {
-    return new Promise(function (resolve, reject) {
```

Comment:
please don't change whitespace without reason
otherwise we can't see any other changes you did.